tempto-examples/docker/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '2'
 services:
   hadoop-master:
     hostname: hadoop-master
-    image: 'ghcr.io/trinodb/testing/hdp2.6-hive:38'
+    image: 'ghcr.io/trinodb/testing/hdp2.6-hive:53'
```

Comment:
This, and jsch update are simplistic. Can you extract them to a separate PR for quick merge?